templates/base.html

```diff
@@ -1,23 +1,41 @@
-<!doctype html>
+{% load static %}
+
+<!DOCTYPE html>
 <html lang="ja">
   <head>
-    <!-- Required meta tags -->
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <title>{% block title %}Twitter Clone{% endblock %}</title>
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
     {% block stylesheet %}{% endblock %}
-    
-    <title>{% block title %}TwitterClone{% endblock title %}</title>
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light mt-3 mb-3">
+      <a class="nav-item nav-link" href="{% url 'tweet:topview' %}">Top</a>
```

Comment:
```suggestion
      <a class="navbar-brand" href="{% url 'tweet:topview' %}">Top</a>
```
公式としてはこちらの書き方が掲載されています。